### ISSUE TITLE
Query Generator Page

### ISSUE THREAD

**Issue by chaukap** [chaukap/capstone_team_4#2]
User Story
--------
As a user, I would like a web form that allows me to customize my differentially private database queries to generate the datasets that suit my needs.

Technical Specifications
--------
**Note that these specifications are minimalist. Other user stories will add additional functionality.**
This page will display the columns of the table specified in #1. For each column, the user will have three options
1) Unique Identifier
2) Grouping
3) Count

A valid submission of this form must satisfy the following conditions:
1) No column can have multiple options selected. 
2) Each option (Unique Identifier, Grouping, and Count) must be selected **exactly once**.

For example, if I had a table

| Uid | Age | Medical Condition | Gender |
| -- | -- | -- | -- |

A valid configuration could be

| Uid | Age | Medical Condition | Gender |
| -- | -- | -- | -- |
| Identifier | Grouping | Count |  |

or

| Uid | Age | Medical Condition | Gender |
| -- | -- | -- | -- |
| Identifier |  | Count | Grouping  |

This information should be sent to the "/query" endpoint of the flask application.

**Comment by chaukap** [comment]
One thing that I forgot to mention on this story- because we aren't storing DB credentials anywhere, they also need to be sent with this request. My thoughts are that the results from #3 will return the credentials supplied as well as the table columns. This page simply needs to send them right back.

TLDR- this page needs to send the same information that is sent in #1 in addition to the selections.
